aptos-move/aptos-vm/src/aptos_vm.rs

```diff
@@ -335,14 +342,44 @@ impl AptosVM {
         let mut session = self
             .vm_impl
             .new_session(resolver, SessionId::epilogue_meta(txn_data));
```

Comment:
I think we should use a different session id for sanity, otherwise we'll forget about it later